### ISSUE TITLE
[Accessibility] Tabs cannot be sent with voice on as the "send" button cannot be selected

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-ios#6312]
## Steps to reproduce
1. Launch Firefox.
1. Login with valid credentials.
1. Access a website.
1. Access the send tab menu.
1. Select a device and then tap on the send button.
    1. Expected behavior
The `send` button is selected and the tab is sent.
    1. Actual behavior
The `Send Tab` button is selected instead of `Send` and the tab cannot be sent.
    1. Device & build information

* Device:  **iPhone X (13.3.1).**
* Build version: **Beta 24.0 (17528).**
    1. Notes
Not reproducible on 12.4 as @asadotzler mentioned on slack.
Attachments: 

[GIF](https://drive.google.com/file/d/1KibOikQyiiEU8Gr1IX-wwH2_a08WPyil/view?usp=sharing)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2316)

**Comment by jcsteh** [comment]
This is a p1 as far as accessibility is concerned. As it is now, it's impossible to use the send tab feature for a VoiceOver user.

**Comment by jcsteh** [comment]
This now works for me on iOS 14.0.1. I'm not sure whether this is somehow specific to iOS 13 or whether something was fixed in Firefox which addresses this regardless of iOS version.

**Comment by thatswinnie** [comment]
@abodea I just tried to reproduce this issue but can't. Could you please check if this bug is still valid or if it can be closed?

**Comment by data-sync-user** [comment]
➤ Sophie Amin commented:

Andrei Bodea Can you check to see if this is still happening on 126? Thank you

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v9000 (42142) with iPhone 15 Pro (17.4.1).
I was able to send a tab without problems.
